packages/web-components/src/radio/README.md

```diff
@@ -0,0 +1,122 @@
+# Radio
+
+> Radio buttons allow users to select a single option from two or more choices. Radio buttons are typically rendered as small circles, which are filled or highlighted when selected.
+
+<br />
+
+## **Design Spec**
+
+[Link to Radio Design Spec in Figma](https://www.figma.com/file/4XWsJrlpEcuEpUnZbtoIBU/Radio?node-id=1295%3A1&t=YOHXLUSK493rMiyh-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 Radio is a form associated component that extends from the FAST Radio [FAST Radio](https://explore.fast.design/components/fast-radio) and is intended to be as close to the Fluent UI React 9 Menu implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+### Use Case
+
+Used anywhere an author might otherwise use an input[type="radio"]. Used to facilitate choice where only one choice is acceptable.
+
+<br />
+
+## Class: `Radio`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name            | Privacy | Type                | Default   | Description                                                                                                                                                                              |
+| --------------- | ------- | ------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `readOnly`      | public  | `boolean`           |           | When true, the control will be immutable by user interaction. See readonly [HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information. |
```

Comment:
This is likely getting removed, we can validate, but I think it's the right move: https://github.com/microsoft/fast/pull/6438